charts/kong/values.yaml

```diff
@@ -358,7 +358,7 @@ ingressController:
     # The annotations for service account
     annotations: {}
 
-  installCRDs: true
```

Comment:
It still does install managed CRDs. The annoying thing with this migration is that there's no good way to ever migrate off it automatically: if you'd ever used the old CRD installation method in the past, you have to continue using it. We can't really discard the old system entirely, so it does still work.

Per the original PR description, Helm doesn't provide a way to remove managed resources from a release. They're still part of it even if you remove the `app.kubernetes.io/managed-by: Helm` label and `meta.helm.sh/release-name` annotation, since Helm tracks them in its release Secret (which isn't designed to allow manual edits) and will delete resources that are there but no longer in template output. There's no good migration path off the legacy system, so I don't think we can deprecate it, even though we don't want any new releases to use it.

We can hide this setting entirely, however, as legacy behavior detection doesn't rely on it being present. Hidden settings are atypical, but this seems like a justifiable case to use them.